src/cgame/cg_local.h

```diff
@@ -1727,8 +1727,8 @@ typedef struct
 	// corpse info
 	clientInfo_t corpseinfo[ MAX_CLIENTS ];
 
-	int          cursorX;
-	int          cursorY;
+	int          cursorX; // unused
```

Comment:
Yes, it is used only in the VM, struct unsafe to be modified are in tr_types.h, qcommon.h, cg_api.h, cg_msgdefs.h and the same for g_